internal/storage/storageutil/control_client.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storageutil
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	control "cloud.google.com/go/storage/control/apiv2"
+	"github.com/googleapis/gax-go/v2"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
+	"google.golang.org/api/option"
+)
+
+func storageControlClientRetryOptions(clientConfig *StorageClientConfig) []gax.CallOption {
+	return []gax.CallOption{gax.WithRetry(func() gax.Retryer {
+		return gax.OnErrorFunc(gax.Backoff{
```

Comment:
I think we should sync with the backend team on retryable codes. I see that they have defined these by default: https://github.com/googleapis/google-cloud-go/blob/storage/v1.40.0/storage/control/apiv2/storage_control_client.go#L79

DeadlineExceeded definitely seems wrong according to the AIP https://google.aip.dev/194

Reply:
To clarify, I will do this offline-- not blocking for this PR.